magma/syntax/transforms/inline_yield_from.py

```diff
@@ -61,14 +61,19 @@ def visit_YieldFrom(self, node):
                     and func.func.value.id == "self"):
                 raise NotImplementedError(ast.dump(func))
             tree = copy.deepcopy(self.method_name_map[func.func.attr])
-            symbol_table = {}
-            # Skip self arg
-            for arg, param in zip(func.args, tree.args.args[1:]):
-                symbol_table[param.arg] = arg
-            tree = replace_symbols(tree, symbol_table)
-            # TODO: Assumes make function is well formed
-            # TODO: Assumes return is well formed
-            return tree.body[0].body[:-1]
+            if (isinstance(tree, ast.FunctionDef) and
+                    tree.name[0:5] == "make_"):
```

Comment:
nit: indentation looks off

Reply:
Style checker forced this indentation (originally had it aligned with the paren, but it complained about same as next logical line and the next line was over indented)